test/cli/run/run-autoinstall.ts

```diff
@@ -0,0 +1,80 @@
+import { expect, test, describe } from "bun:test";
+import { mkdirSync } from "fs";
+import { bunEnv, bunExe, tmpdirSync } from "harness";
+import { join } from "path";
+
+//   --install=<val>                 Configure auto-install behavior. One of "auto" (default, auto-installs when no node_modules), "fallback" (missing packages only), "force" (always).
+//   -i                              Auto-install dependencies during execution. Equivalent to --install=fallback.
+
+describe("basic autoinstall", () => {
```

Comment:
can you make this filename end with .test.ts? as is, this doesn't run in CI

Reply:
oh yeah I somehow forgot, sorry